react/Gallery.js

```diff
@@ -57,8 +58,9 @@ class Gallery extends Component {
     const from = index * itemsPerRow
     const rowItems = products.slice(from, from + itemsPerRow)
 
-    const containerClasses = classNames(searchResult.galleryRow, {
+    const containerClasses = classNames(searchResult.galleryRow, 'flex', {
       [searchResult.galleryTwoColumns]: layoutMode === 'small' && mobile,
+      'justify-center': mobile,
```

Comment:
Why only on mobile? The cells are not centered in the desktop version? 
![image](https://user-images.githubusercontent.com/18706156/52227283-3cb1c700-2896-11e9-8225-569e05894faa.png)

Reply:
Design in web mode aligns left.

Reply:
We need to involve @oldufre is this discussion to not have conflicting informations about it.